docs/pages/application-access/okta/scim-only.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing SCIM-only Okta integration
+description: How to install a SCIM-only Okta integration
+---
+
```

Comment:
We should include an intro paragraph here so users know whether this guide is appropriate for them.